docs/UserGuide.adoc

```diff
@@ -372,25 +445,25 @@ View various statistics about the memes such as tags and likes count by tags, an
 being the most recommended one.
 * For the command word suggestion, only commands available for current context will be displayed.
 Description for each command will be displayed after each command word.
-* User can press "TAB" key to auto-complete the command suggestion, i.e. replacing current command word/arguments with
+* User can press kbd:[Tab] key to auto-complete the command suggestion, i.e. replacing current command word/arguments with
 the first suggestion displayed (if there is any).
 * If the user input is of invalid format, the text will turn red and error messages will be displayed in the result box
-immediately without pressing "ENTER" key. This does not account for invalid values, e.g. input meme index is 5 but there is
+immediately without pressing kbd:[Enter] key. This does not account for invalid values, e.g. input meme index is 5 but there is
 no meme of index 5.
 
 Example 1: +
 When user type in "a" in the meme context, the following suggestions will appear: +
 `add: adds a meme to Weme. +
 archive: archive a meme by index. +
 archives: list all archived memes.` +
-If tab is pressed, "a" in the command box will be replaced by "add".
+If kbd:[Tab] is pressed, "a" in the command box will be replaced by "add".
 
 Example 2: +
 when user type in "add p/pathToMeme t/c", the following suggestions will appear: +
```

Comment:
When not when

Reply:
user types not user type. Also types not type in point 2 (not displayed in this change)